src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-field.html

```diff
@@ -1,80 +1,149 @@
-<!-- Debugging: html_theme_options = {{ html_theme_options }} -->
-{% if theme_use_meilisearch %}
-  {% set API_KEY = theme_use_meilisearch.api_key %}
-  <!-- Include the MeiliSearch CSS stylesheet only if theme present -->
-  <link href="{{ pathto('_static/css/meilisearch.css', 1) }}" rel="stylesheet" />
-  <!-- Create a search bar form with a text input and a dropdown selector for the search index -->
-  <form class="bd-search d-flex align-items-center"
-  action="{{ pathto('search') }}"
-  method="get">
+<form class="bd-search d-flex align-items-center" action="{{ pathto('search') }}" method="get">
   <i class="fa-solid fa-magnifying-glass"></i>
-  <input type="search"
-    class="form-control"
-    name="q"
-    placeholder="{{ theme_search_bar_text }}"
-    aria-label="{{ theme_search_bar_text }}"
-    autocomplete="off"
-    autocorrect="off"
-    autocapitalize="off"
-    spellcheck="false"/>
+  <input type="search" class="form-control" name="q" placeholder="{{ theme_search_bar_text }}" aria-label="{{ theme_search_bar_text }}" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
   <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
-  <!-- Conditionally display the dropdown selector if there are multiple indices -->
-  {% if theme_use_meilisearch.index_uids | length > 1 %}
-    <select id="indexUidSelector" class="index-select">
-    {% for index_uid, index_name in theme_use_meilisearch.index_uids.items() %}
-      <option value="{{ index_uid }}">{{ index_name }}</option>
-    {% endfor %}
-    </select>
-    <style>
-      .form-control:focus,.form-control:focus-visible,.form-control {
-        width: 490px;
+</form>
+
+<!-- Add this div to show the search results -->
+<div id="results" class="results"></div>
+
+<link href="{{ pathto('_static/css/fuse-search.css', 1) }}" rel="stylesheet" />
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js"></script>
+
+
+<!-- Script tag for search functionality -->
+ {% set theme_static_search = theme_static_search | default({}) %}
+  {% set threshold = theme_static_search.threshold | default(0.5) %}
+  {% set use_extended_search = theme_static_search.use_extended_search | default(true) %}
+  {% set should_sort = theme_static_search.should_sort | default(true) %}
+  {% set ignoreLocation = theme_static_search.ignoreLocation | default(false) %}
+  {% set limit = theme_static_search.docs_limit | default(20) %}
+<script>
+  require.config({
+    paths: {
+      fuse: "https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min"
+    }
+  });
+
+  require(['fuse'], function(Fuse) {
+    let fuseInstance;
+    let searchData = [];
+
+    function initializeFuse(data) {
+      const fuseOptions = {
+        keys: ['title', 'text'],
+        threshold: "{{ threshold }}",
```

Comment:
Are these options coming from the `conf.py`?

Reply:
yes, it is from conf.py -> html theme options - static_search dict

Reply:
Thanks!